cryptohash-sha512.cabal

```diff
@@ -41,7 +41,7 @@ source-repository head
 
 library
   default-language:  Haskell2010
-  build-depends:     base             >= 4.5   && < 4.10
+  build-depends:     base             >= 4.5   && < 4.11
```

Comment:
I think you rather intended `< 4.12`; c.f. http://hackage.haskell.org/package/cryptohash-sha512-0.11.100.1/revisions/

Reply:
I didn't have time to test with 4.11 and since that has had a fair amount of breakages I decided to be conservative.